experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -917,6 +950,22 @@ def test_end_date_uses_cached_end_date(self):
     def test_enrollment_duration_for_ended_experiment(self):
         experiment = NimbusExperimentFactory.create_with_lifecycle(
             NimbusExperimentFactory.Lifecycles.LIVE_ENROLLING,
+            is_first_run=True,
+            proposed_release_date=datetime.date.today() + datetime.timedelta(days=1),
+        )
+
+        assert experiment.proposed_release_date
+        assert experiment.computed_end_date
```

Comment:
don't we want to assert with something? 🤔

Reply:
same ^ I'm happy to take these out, I was just following what was previously there